src/wstore/store_commons/utils/tests.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017 CoNWeT Lab., Universidad Politécnica de Madrid
+
+# This file belongs to the business-charging-backend
+# of the Business API Ecosystem.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.test import TestCase
+
+from wstore.store_commons.utils.units import ChargePeriod, CurrencyCode
+
+
+class ChargePeriodUnitsTestCase(TestCase):
+
+    tags = ('units',)
+
+    def setUp(self):
+        self.cp_valid = {
+            'title': 'daily',
+            'value': 1,
+        }
+        self.cp_not_valid = {
+            'title': 'weekly',
+            'value': 7,
+        }
+
+        settings.CHARGE_PERIODS = [
```

Comment:
Don't override the global settings for the test, you can use the decorator @override_settings in the different tests in order to have this behaviour